### PR TITLE
Update customer-lockbox-requests.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a FAQ on how to setup alerts when customer lockbox is approved.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5717

File: microsoft-365/compliance/customer-lockbox-requests.md [hunk 1]
```diff
@@ -255,6 +255,10 @@ Customer Lockbox requests have a default duration of 12 hours. If you don't resp
 
 If a customer rejects a Customer Lockbox request, no access to customer content occurs. If a user in your organization continues to experience a service issue requiring Microsoft to access customer content to resolve the issue, then the service issue might persist and Microsoft will inform the user about this.
 
+#### How to setup Alerts any time a request has been approved ?
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-lockbox-requests.md]:
```suggestion
#### How do I set up alerts whenever a request has been approved?
```

File: microsoft-365/compliance/customer-lockbox-requests.md [hunk 2]
```diff
@@ -255,6 +255,10 @@ Customer Lockbox requests have a default duration of 12 hours. If you don't resp
...
+
+There is no built in option to Alert the administrators. However, Administrators can setup Alerts using [Microsoft cloud App security](/cloud-app-security/getting-started-with-cloud-app-security#to-create-policies).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-lockbox-requests.md]:
```suggestion
There is no built-in option to alert administrators. However, administrators can set up alerts using [Microsoft cloud App security](/cloud-app-security/getting-started-with-cloud-app-security#to-create-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder

**Pull request merged**